assets/css/style.css

```diff
@@ -0,0 +1,101 @@
+.container {
+  max-width: 375px;
+  height: 652px;
+  border: 1px grey solid;
```

Comment:
- [ ] Don't forget to take out this border as it is not in the template. I think it was used as a guide, which is ok 😺

Reply:
It's done, the border was used just to reference the size of the mobile.